Lib/idlelib/config.py

```diff
@@ -192,7 +193,7 @@ def GetUserCfgDir(self):
                     except OSError:
                         pass
                 userDir = '~'
-        if userDir == "~": # still no path to home!
+        if userDir == '~': # still no path to home!
```

Comment:
Avoid making unnecessary changes that are unrelated to your functional change

Reply:
Reverted unnecessary formatting changes in dc46fb7